frontside/lib/screens/add_spot_screen.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+
+import '../generated/l10n.dart';
+import '../widgets/index.dart';
+
+class AddSpotScreen extends StatelessWidget {
+  static String route = '/add-spot';
+  const AddSpotScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return AppScaffold(
+      title: AppI18n.of(context).addSpot,
```

Comment:
Even though the text is the same, the title should get its own property `addSpotTitle`.